misc/dist/linux/org.godotengine.Godot.desktop

```
@@ -4,11 +4,13 @@ GenericName=Libre game engine
 GenericName[el]=Ελεύθερη μηχανή παιχνιδιού
 GenericName[fr]=Moteur de jeu libre
 GenericName[nl]=Libre game-engine
+GenericName[ru]=Свободный игровой движок
 GenericName[zh_CN]=自由的游戏引擎
 Comment=Multi-platform 2D and 3D game engine with a feature-rich editor
 Comment[el]=2D και 3D μηχανή παιχνιδιού πολλαπλών πλατφορμών με επεξεργαστή πλούσιο σε χαρακτηριστικά
 Comment[fr]=Moteur de jeu 2D et 3D multiplateforme avec un éditeur riche en fonctionnalités
 Comment[nl]=Multi-platform 2D- en 3d-game-engine met een veelzijdige editor
```

```suggestion
Comment[nl]=Multi-platform 2D- en 3D-game-engine met een veelzijdige editor
```

I don't know Dutch, but it looks like an obvious typo.